lib/zendesk_api/silent_mash.rb

```diff
@@ -0,0 +1,8 @@
+require 'hashie'
+
+module ZendeskAPI
+  # @private
+  class SilentMash < Hashie::Mash
+    disable_warnings
```

Comment:
update your hashie ... but I guess should bump required version here and yank the previous release ...